config/locales/pt-BR.yml

```
@@ -329,13 +329,14 @@ pt-BR:
   sessions:
     new:
       forgot_password: Esqueceu sua Senha?
-      resend_confirmation:
+      resend_confirmation: Não recebi o email de confirmação?
```

```suggestion
      resend_confirmation: Não recebeu o email de confirmação?
```

This would be more inline with what the english translation says.

---

Sure, totally agree. Thanks for your review.